test/common/testlib.py

```diff
@@ -1005,6 +1005,11 @@ def assert_pixels_in_current_layout(self, selector: str, key: str,
         if ignore is None:
             ignore = []
 
+        # Let's ignore PF SVG icons, they seem to be not entirely
+        # pixel perfect
+
+        ignore = ignore + [".pf-v5-svg"]
```

Comment:
If it's easy I would prefer to allow a small percentage of change, rather than skip these. I have an example where pixel tests on icons are super useful. https://github.com/cockpit-project/cockpit/pull/18822

Reply:
If not easy obviously let's not spent much time on this.

Reply:
I also see this a lot on https://github.com/cockpit-project/cockpit/pull/18823

Reply:
Good point, thanks!

Reply:
Nevermind, my PR has a whole different set of issues :)